api/src/main/java/org/apache/cloudstack/api/response/BucketResponse.java

```diff
@@ -98,7 +98,7 @@ public class BucketResponse extends BaseResponseWithTagInformation implements Co
     @Param(description = "Bucket Access Key")
     private String accessKey;
 
-    @SerializedName(ApiConstants.SECRET_KEY)
+    @SerializedName(ApiConstants.USER_SECRET_KEY)
```

Comment:
same here, but not as important as previous one

Reply:
same as https://github.com/apache/cloudstack/pull/8674#discussion_r1636187875